EDA.qmd

```diff
@@ -213,505 +214,508 @@ unusual
 
 ```{r}
 #| message: false
 #| dev: "png"
 #| fig-alt: |
-#|   A scatterplot of residuals vs. carat of diamonds. The x-axis ranges from 0 
-#|   to 5, the y-axis ranges from 0 to almost 4. Much of the data are clustered 
-#|   around low values of carat and residuals. There is a clear, curved pattern 
-#|   showing decrease in residuals as carat increases.
+#| Um gráfico de dispersão de resíduos versus quilate de diamantes. O eixo x varia de 0
+#| a 5, o eixo y varia de 0 a quase 4. Muitos dos dados estão agrupados
+#| em torno de valores baixos de quilate e resíduos. Há um padrão claro e curvo
+#| mostrando diminuição nos resíduos à medida que o quilate aumenta.
 
 library(tidymodels)
 
-diamonds <- diamonds |>
+diamante <- diamante |>
   mutate(
-    log_price = log(price),
-    log_carat = log(carat)
+    log_preco = log(preco),
+    log_quilate = log(quilate)
   )
 
-diamonds_fit <- linear_reg() |>
-  fit(log_price ~ log_carat, data = diamonds)
+diamante_fit <- linear_reg() |>
+  fit(log_preco ~ log_quilate, data = diamante)
 
-diamonds_aug <- augment(diamonds_fit, new_data = diamonds) |>
+diamante_aug <- augment(diamante_fit, new_data = diamante) |>
   mutate(.resid = exp(.resid))
 
-ggplot(diamonds_aug, aes(x = carat, y = .resid)) + 
+ggplot(diamante_aug, aes(x = quilate, y = .resid)) + 
   geom_point()
 ```
 
-Once you've removed the strong relationship between carat and price, you can see what you expect in the relationship between cut and price: relative to their size, better quality diamonds are more expensive.
+Uma vez que você tem removido a forte relação entre quilate e preço, você pode ver o esperado da relação entre corte e preço: relativo ao tamanho deles, diamantes de melhor qualidade (corte) são mais caros.
 
 ```{r}
 #| fig-alt: |
-#|   Side-by-side box plots of residuals by cut. The x-axis displays the various 
-#|   cuts (Fair to Ideal), the y-axis ranges from 0 to almost 5. The medians are 
-#|   quite similar, between roughly 0.75 to 1.25. Each of the distributions of 
-#|   residuals is right skewed, with many outliers on the higher end.
+#| Gráficos boxplot lado a lado de resíduos por corte. O eixo x exibe os vários
+#| cortes (justo para ideal), o eixo y varia de 0 a quase 5. As medianas são
+#| bastante semelhantes, entre cerca de 0,75 a 1,25. Cada uma das distribuições 
+#| dos resíduos são distorcidos para a direita, com muitos valores discrepantes na
+#| extremidade superior.
 
-ggplot(diamonds_aug, aes(x = cut, y = .resid)) + 
+ggplot(diamante_aug, aes(x = corte, y = .resid)) + 
   geom_boxplot()
 ```
 
-We're not discussing modelling in this book because understanding what models are and how they work is easiest once you have tools of data wrangling and programming in hand.
+Nós não discutimos modelagem nesse livro porque entender o que são os modelos e como eles trabalham é mais fácil uma vez que você tem ferramentas de organização de dados (data wrangling) e programação em mãos.
```

Comment:
```suggestion
Nós não discutiremos modelagem nesse livro porque será mais fácil entender o que são os modelos e como eles funcionam uma vez que você tenha ferramentas de tratamento de dados (data wrangling) e de programação em mãos.
```
achei q fluía melhor em português nessa ordem

Reply:
Eu formataria data wrangling em itálico.